cmd/register/register_util.go

```diff
@@ -276,14 +307,18 @@ func hydrateLaunchPlanSpec(lpSpec *admin.LaunchPlanSpec) {
 			OutputLocationPrefix: rconfig.DefaultFilesConfig.OutputLocationPrefix,
```

Comment:
Need to refactor `rconfig.DefaultFilesConfig.OutputLocationPrefix`. We can't use directly as per new change

Reply:
Done.Rebased